### PR TITLE
[WIP] Issue #1369 Fixing TestCacheOc unit test

### LINKED ISSUE #1369: TestCacheOc unit test is failing intermittently
```
=== RUN   TestCacheOc
--- FAIL: TestCacheOc (1.11s)
	oc_caching_test.go:77: Error attempting to download and cache oc: File has no hash to validate - not downloading
FAIL
FAIL	github.com/minishift/minishift/pkg/minishift/cache	1.132s
```
Example: 
* https://circleci.com/gh/minishift/minishift/2030?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
* https://ci.appveyor.com/project/minishift-bot/minishift/build/557

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Fixes #1369 

Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

**Comment by gbraad** [comment]
Doesn't seem to fix it on TravisCI
```
--- FAIL: TestCacheOc (1.07s)
	oc_caching_test.go:77: Error attempting to download and cache oc: File has no hash to validate - not downloading
FAIL
```

**Comment by LalatenduMohanty** [comment]
@gbraad right. I need to dig more.

File: pkg/minishift/cache/oc_caching_test.go [hunk 1]
```diff
@@ -99,7 +99,7 @@ func addMockResponses(mockTransport *minitesting.MockRoundTripper) {
 	mockTransport.RegisterResponse("https://api.github.com/repos/openshift/origin/releases/assets/2489308", &minitesting.CannedResponse{
 		ResponseType: minitesting.SERVE_FILE,
 		Response:     filepath.Join(testDataDir, "CHECKSUM"),
-		ContentType:  minitesting.TEXT,
+		ContentType:  minitesting.JSON,
```
Comment (praveenkumar) [inline pkg/minishift/cache/oc_caching_test.go]: CHECKSUM content type is `TEXT` so then MIME type here is correct. No change required here.

**Comment by praveenkumar** [comment]
Closing this because surpass by #1413

**Pull request closed without merging**